@coderabbitai Fix test code, console logs gave

⎯⎯⎯⎯⎯⎯⎯ Failed Tests 3 ⎯⎯⎯⎯⎯⎯⎯

 FAIL  src/components/tierList/TierLists.test.tsx > TierLists > renders controls, distribution bar, disclaimer and board once categories resolve
TestingLibraryElementError: Unable to find an element by: [data-testid="toggle-sort"]

Ignored nodes: comments, script, style
<body>
  <div>
    <div
      class="MuiBox-root css-au904d"
    >
      <button
        class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-sizeMedium MuiButton-colorPrimary css-1blvszr-MuiButtonBase-root-MuiButton-root"
        tabindex="0"
        type="button"
      >
        <span
          class="MuiButton-icon MuiButton-startIcon css-1lxyhll-MuiButton-startIcon"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1umw9bq-MuiSvgIcon-root"
            data-testid="SwapVertIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M16 17.01V10h-2v7.01h-3L15 21l4-3.99zM9 3 5 6.99h3V14h2V6.99h3z"
            />
          </svg>
        </span>
      </button>
    </div>
    <div
      class="MuiBox-root css-1b46un8"
    >
      <div
        class="MuiBox-root css-3mllz6"
      >
        <p
          class="MuiTypography-root MuiTypography-body2 css-1a40gj6-MuiTypography-root"
        >
          Distribution
        </p>
        <p
          class="MuiTypography-root MuiTypography-body2 css-7ctihp-MuiTypography-root"
        >
          1 games
        </p>
      </div>
      <div
        class="MuiBox-root css-1ka3ow9"
      >
        <div
          aria-label="tooltip"
          class="MuiBox-root css-jti6s"
          data-mui-internal-clone-element="true"
          data-testid="distribution-segment-tier_good"
        />
      </div>
    </div>
    <div
      class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiAccordion-root MuiAccordion-rounded MuiAccordion-gutters css-1808mag-MuiPaper-root-MuiAccordion-root"
      style="--Paper-shadow: 0px 2px 1px -1px rgba(0,0,0,0.2),0px 1px 1px 0px rgba(0,0,0,0.14),0px 1px 3px 0px rgba(0,0,0,0.12);"
    >
      <h3
        class="MuiAccordion-heading css-cy7rkm-MuiAccordion-heading"
      >
        <button
          aria-controls="tier-list-disclaimer-content"
          aria-expanded="false"
          class="MuiButtonBase-root MuiAccordionSummary-root MuiAccordionSummary-gutters css-1f1pi2l-MuiButtonBase-root-MuiAccordionSummary-root"
          id="tier-list-disclaimer-header"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiAccordionSummary-content css-yfrx4k-MuiAccordionSummary-content"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-colorInfo MuiSvgIcon-fontSizeMedium css-kswx6x-MuiSvgIcon-root"
              data-testid="HelpIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2m1 17h-2v-2h2zm2.07-7.75-.9.92C13.45 12.9 13 13.5 13 15h-2v-.5c0-1.1.45-2.1 1.17-2.83l1.24-1.26c.37-.36.59-.86.59-1.41 0-1.1-.9-2-2-2s-2 .9-2 2H8c0-2.21 1.79-4 4-4s4 1.79 4 4c0 .88-.36 1.68-.93 2.25"
              />
            </svg>
            <p
              class="MuiTypography-root MuiTypography-body1 css-rizt0-MuiTypography-root"
            >
              Disclaimer
            </p>
          </span>
          <span
            class="MuiAccordionSummary-expandIconWrapper css-1wqf3nl-MuiAccordionSummary-expandIconWrapper"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1umw9bq-MuiSvgIcon-root"
              data-testid="ExpandMoreIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M16.59 8.59 12 13.17 7.41 8.59 6 10l6 6 6-6z"
              />
            </svg>
          </span>
        </button>
      </h3>
      <div
        class="MuiCollapse-root MuiCollapse-vertical MuiCollapse-hidden css-cwrbtg-MuiCollapse-root"
        style="min-height: 0px;"
      >
        <div
          class="MuiCollapse-wrapper MuiCollapse-vertical css-1x6hinx-MuiCollapse-wrapper"
        >
          <div
            class="MuiCollapse-wrapperInner MuiCollapse-vertical css-1i4ywhz-MuiCollapse-wrapperInner"
          >
            <div
              aria-labelledby="tier-list-disclaimer-header"
              class="MuiAccordion-region css-at4ued-MuiAccordion-region"
              id="tier-list-disclaimer-content"
              role="region"
            >
              <div
                class="MuiAccordionDetails-root css-1lj39kh-MuiAccordionDetails-root"
              >
                <p
                  class="MuiTypography-root MuiTypography-body1 css-rizt0-MuiTypography-root"
                >
                  text
                </p>
                <div
                  class="MuiBox-root css-1yuhvjn"
                >
                  <ul
                    class="MuiList-root css-54zhjy-MuiList-root"
                  >
                    <li
                      class="MuiListItem-root MuiListItem-gutters MuiListItem-padding css-cfd5wn-MuiListItem-root"
              ...
 ❯ Object.getElementError node_modules/@testing-library/dom/dist/config.js:37:19
 ❯ node_modules/@testing-library/dom/dist/query-helpers.js:76:38
 ❯ node_modules/@testing-library/dom/dist/query-helpers.js:52:17
 ❯ node_modules/@testing-library/dom/dist/query-helpers.js:95:19
 ❯ src/components/tierList/TierLists.test.tsx:123:23
    121|
    122|         // Controls (mocked) are present
    123|         expect(screen.getByTestId('toggle-sort')).toBeInTheDocument();
       |                       ^
    124|
    125|         // DistributionHeader is shown (our next-intl mock provides "D…

⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯[1/3]⎯

 FAIL  src/components/tierList/TierLists.test.tsx > TierLists > toggles sortOrder from asc to desc when the controls are used
TestingLibraryElementError: Unable to find an element by: [data-testid="toggle-sort"]

Ignored nodes: comments, script, style
<body>
  <div>
    <div
      class="MuiBox-root css-au904d"
    >
      <button
        class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-sizeMedium MuiButton-colorPrimary css-1blvszr-MuiButtonBase-root-MuiButton-root"
        tabindex="0"
        type="button"
      >
        <span
          class="MuiButton-icon MuiButton-startIcon css-1lxyhll-MuiButton-startIcon"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1umw9bq-MuiSvgIcon-root"
            data-testid="SwapVertIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M16 17.01V10h-2v7.01h-3L15 21l4-3.99zM9 3 5 6.99h3V14h2V6.99h3z"
            />
          </svg>
        </span>
      </button>
    </div>
    <div
      class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiAccordion-root MuiAccordion-rounded MuiAccordion-gutters css-1808mag-MuiPaper-root-MuiAccordion-root"
      style="--Paper-shadow: 0px 2px 1px -1px rgba(0,0,0,0.2),0px 1px 1px 0px rgba(0,0,0,0.14),0px 1px 3px 0px rgba(0,0,0,0.12);"
    >
      <h3
        class="MuiAccordion-heading css-cy7rkm-MuiAccordion-heading"
      >
        <button
          aria-controls="tier-list-disclaimer-content"
          aria-expanded="false"
          class="MuiButtonBase-root MuiAccordionSummary-root MuiAccordionSummary-gutters css-1f1pi2l-MuiButtonBase-root-MuiAccordionSummary-root"
          id="tier-list-disclaimer-header"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiAccordionSummary-content css-yfrx4k-MuiAccordionSummary-content"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-colorInfo MuiSvgIcon-fontSizeMedium css-kswx6x-MuiSvgIcon-root"
              data-testid="HelpIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2m1 17h-2v-2h2zm2.07-7.75-.9.92C13.45 12.9 13 13.5 13 15h-2v-.5c0-1.1.45-2.1 1.17-2.83l1.24-1.26c.37-.36.59-.86.59-1.41 0-1.1-.9-2-2-2s-2 .9-2 2H8c0-2.21 1.79-4 4-4s4 1.79 4 4c0 .88-.36 1.68-.93 2.25"
              />
            </svg>
            <p
              class="MuiTypography-root MuiTypography-body1 css-rizt0-MuiTypography-root"
            >
              Disclaimer
            </p>
          </span>
          <span
            class="MuiAccordionSummary-expandIconWrapper css-1wqf3nl-MuiAccordionSummary-expandIconWrapper"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1umw9bq-MuiSvgIcon-root"
              data-testid="ExpandMoreIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M16.59 8.59 12 13.17 7.41 8.59 6 10l6 6 6-6z"
              />
            </svg>
          </span>
        </button>
      </h3>
      <div
        class="MuiCollapse-root MuiCollapse-vertical MuiCollapse-hidden css-cwrbtg-MuiCollapse-root"
        style="min-height: 0px;"
      >
        <div
          class="MuiCollapse-wrapper MuiCollapse-vertical css-1x6hinx-MuiCollapse-wrapper"
        >
          <div
            class="MuiCollapse-wrapperInner MuiCollapse-vertical css-1i4ywhz-MuiCollapse-wrapperInner"
          >
            <div
              aria-labelledby="tier-list-disclaimer-header"
              class="MuiAccordion-region css-at4ued-MuiAccordion-region"
              id="tier-list-disclaimer-content"
              role="region"
            >
              <div
                class="MuiAccordionDetails-root css-1lj39kh-MuiAccordionDetails-root"
              >
                <p
                  class="MuiTypography-root MuiTypography-body1 css-rizt0-MuiTypography-root"
                >
                  text
                </p>
                <div
                  class="MuiBox-root css-1yuhvjn"
                >
                  <ul
                    class="MuiList-root css-54zhjy-MuiList-root"
                  >
                    <li
                      class="MuiListItem-root MuiListItem-gutters MuiListItem-padding css-cfd5wn-MuiListItem-root"
                    >
                      <div
                        class="MuiListItemAvatar-root css-1ulznza-MuiListItemAvatar-root"
                      >
                        <div
                          class="MuiAvatar-root MuiAvatar-circular MuiAvatar-colorDefault css-1h5o4ln-MuiAvatar-root"
                        >
                          <svg
                            aria-hidden="true"
                            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1umw9bq-MuiSvgIcon-root"
                            data-testid="SportsEsportsIcon"
                            focusable="false"
                            viewBox="0 0 24 24"
                          >
                            <path
                              d="m21.58 16.09-1.09-7.66C20.21 6.46 18.52 5 16.53 5H7.47C5.48 5 3.79...
 ❯ Object.getElementError node_modules/@testing-library/dom/dist/config.js:37:19
 ❯ node_modules/@testing-library/dom/dist/query-helpers.js:76:38
 ❯ node_modules/@testing-library/dom/dist/query-helpers.js:52:17
 ❯ node_modules/@testing-library/dom/dist/query-helpers.js:95:19
 ❯ src/components/tierList/TierLists.test.tsx:160:32
    158|         });
    159|         render(<TierLists GameRender={GameRender} />);
    160|         fireEvent.click(screen.getByTestId('toggle-sort'));
       |                                ^
    161|         expect(useGetSortedCategoriesQueryMock).toHaveBeenLastCalledWi…
    162|     });

⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯[2/3]⎯

 FAIL  src/components/tierList/TierLists.test.tsx > TierLists > toggles back to asc on a second click
TestingLibraryElementError: Unable to find an element by: [data-testid="toggle-sort"]

Ignored nodes: comments, script, style
<body>
  <div>
    <div
      class="MuiBox-root css-au904d"
    >
      <button
        class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-sizeMedium MuiButton-colorPrimary css-1blvszr-MuiButtonBase-root-MuiButton-root"
        tabindex="0"
        type="button"
      >
        <span
          class="MuiButton-icon MuiButton-startIcon css-1lxyhll-MuiButton-startIcon"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1umw9bq-MuiSvgIcon-root"
            data-testid="SwapVertIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M16 17.01V10h-2v7.01h-3L15 21l4-3.99zM9 3 5 6.99h3V14h2V6.99h3z"
            />
          </svg>
        </span>
      </button>
    </div>
    <div
      class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiAccordion-root MuiAccordion-rounded MuiAccordion-gutters css-1808mag-MuiPaper-root-MuiAccordion-root"
      style="--Paper-shadow: 0px 2px 1px -1px rgba(0,0,0,0.2),0px 1px 1px 0px rgba(0,0,0,0.14),0px 1px 3px 0px rgba(0,0,0,0.12);"
    >
      <h3
        class="MuiAccordion-heading css-cy7rkm-MuiAccordion-heading"
      >
        <button
          aria-controls="tier-list-disclaimer-content"
          aria-expanded="false"
          class="MuiButtonBase-root MuiAccordionSummary-root MuiAccordionSummary-gutters css-1f1pi2l-MuiButtonBase-root-MuiAccordionSummary-root"
          id="tier-list-disclaimer-header"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiAccordionSummary-content css-yfrx4k-MuiAccordionSummary-content"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-colorInfo MuiSvgIcon-fontSizeMedium css-kswx6x-MuiSvgIcon-root"
              data-testid="HelpIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2m1 17h-2v-2h2zm2.07-7.75-.9.92C13.45 12.9 13 13.5 13 15h-2v-.5c0-1.1.45-2.1 1.17-2.83l1.24-1.26c.37-.36.59-.86.59-1.41 0-1.1-.9-2-2-2s-2 .9-2 2H8c0-2.21 1.79-4 4-4s4 1.79 4 4c0 .88-.36 1.68-.93 2.25"
              />
            </svg>
            <p
              class="MuiTypography-root MuiTypography-body1 css-rizt0-MuiTypography-root"
            >
              Disclaimer
            </p>
          </span>
          <span
            class="MuiAccordionSummary-expandIconWrapper css-1wqf3nl-MuiAccordionSummary-expandIconWrapper"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1umw9bq-MuiSvgIcon-root"
              data-testid="ExpandMoreIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M16.59 8.59 12 13.17 7.41 8.59 6 10l6 6 6-6z"
              />
            </svg>
          </span>
        </button>
      </h3>
      <div
        class="MuiCollapse-root MuiCollapse-vertical MuiCollapse-hidden css-cwrbtg-MuiCollapse-root"
        style="min-height: 0px;"
      >
        <div
          class="MuiCollapse-wrapper MuiCollapse-vertical css-1x6hinx-MuiCollapse-wrapper"
        >
          <div
            class="MuiCollapse-wrapperInner MuiCollapse-vertical css-1i4ywhz-MuiCollapse-wrapperInner"
          >
            <div
              aria-labelledby="tier-list-disclaimer-header"
              class="MuiAccordion-region css-at4ued-MuiAccordion-region"
              id="tier-list-disclaimer-content"
              role="region"
            >
              <div
                class="MuiAccordionDetails-root css-1lj39kh-MuiAccordionDetails-root"
              >
                <p
                  class="MuiTypography-root MuiTypography-body1 css-rizt0-MuiTypography-root"
                >
                  text
                </p>
                <div
                  class="MuiBox-root css-1yuhvjn"
                >
                  <ul
                    class="MuiList-root css-54zhjy-MuiList-root"
                  >
                    <li
                      class="MuiListItem-root MuiListItem-gutters MuiListItem-padding css-cfd5wn-MuiListItem-root"
                    >
                      <div
                        class="MuiListItemAvatar-root css-1ulznza-MuiListItemAvatar-root"
                      >
                        <div
                          class="MuiAvatar-root MuiAvatar-circular MuiAvatar-colorDefault css-1h5o4ln-MuiAvatar-root"
                        >
                          <svg
                            aria-hidden="true"
                            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1umw9bq-MuiSvgIcon-root"
                            data-testid="SportsEsportsIcon"
                            focusable="false"
                            viewBox="0 0 24 24"
                          >
                            <path
                              d="m21.58 16.09-1.09-7.66C20.21 6.46 18.52 5 16.53 5H7.47C5.48 5 3.79...
 ❯ Object.getElementError node_modules/@testing-library/dom/dist/config.js:37:19
 ❯ node_modules/@testing-library/dom/dist/query-helpers.js:76:38
 ❯ node_modules/@testing-library/dom/dist/query-helpers.js:52:17
 ❯ node_modules/@testing-library/dom/dist/query-helpers.js:95:19
 ❯ src/components/tierList/TierLists.test.tsx:172:32
    170|         });
    171|         render(<TierLists GameRender={GameRender} />);
    172|         fireEvent.click(screen.getByTestId('toggle-sort'));
       |                                ^
    173|         fireEvent.click(screen.getByTestId('toggle-sort'));
    174|         expect(useGetSortedCategoriesQueryMock).toHaveBeenLastCalledWi…

⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯[3/3]⎯


 Test Files  1 failed | 101 passed (102)
      Tests  3 failed | 528 passed (531)
   Start at  21:20:01
   Duration  54.71s (transform 2.61s, setup 12.23s, import 15.93s, tests 10.24s, environment 95.15s)


Error: TestingLibraryElementError: Unable to find an element by: [data-testid="toggle-sort"]

Ignored nodes: comments, script, style
<body>
  <div>
    <div
      class="MuiBox-root css-au904d"
    >
      <button
        class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-sizeMedium MuiButton-colorPrimary css-1blvszr-MuiButtonBase-root-MuiButton-root"
        tabindex="0"
        type="button"
      >
        <span
          class="MuiButton-icon MuiButton-startIcon css-1lxyhll-MuiButton-startIcon"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1umw9bq-MuiSvgIcon-root"
            data-testid="SwapVertIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M16 17.01V10h-2v7.01h-3L15 21l4-3.99zM9 3 5 6.99h3V14h2V6.99h3z"
            />
          </svg>
        </span>
      </button>
    </div>
    <div
      class="MuiBox-root css-1b46un8"
    >
      <div
        class="MuiBox-root css-3mllz6"
      >
        <p
          class="MuiTypography-root MuiTypography-body2 css-1a40gj6-MuiTypography-root"
        >
          Distribution
        </p>
        <p
          class="MuiTypography-root MuiTypography-body2 css-7ctihp-MuiTypography-root"
        >
          1 games
        </p>
      </div>
      <div
        class="MuiBox-root css-1ka3ow9"
      >
        <div
          aria-label="tooltip"
          class="MuiBox-root css-jti6s"
          data-mui-internal-clone-element="true"
          data-testid="distribution-segment-tier_good"
        />
      </div>
    </div>
    <div
      class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiAccordion-root MuiAccordion-rounded MuiAccordion-gutters css-1808mag-MuiPaper-root-MuiAccordion-root"
      style="--Paper-shadow: 0px 2px 1px -1px rgba(0,0,0,0.2),0px 1px 1px 0px rgba(0,0,0,0.14),0px 1px 3px 0px rgba(0,0,0,0.12);"
    >
      <h3
        class="MuiAccordion-heading css-cy7rkm-MuiAccordion-heading"
      >
        <button
          aria-controls="tier-list-disclaimer-content"
          aria-expanded="false"
          class="MuiButtonBase-root MuiAccordionSummary-root MuiAccordionSummary-gutters css-1f1pi2l-MuiButtonBase-root-MuiAccordionSummary-root"
          id="tier-list-disclaimer-header"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiAccordionSummary-content css-yfrx4k-MuiAccordionSummary-content"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-colorInfo MuiSvgIcon-fontSizeMedium css-kswx6x-MuiSvgIcon-root"
              data-testid="HelpIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2m1 17h-2v-2h2zm2.07-7.75-.9.92C13.45 12.9 13 13.5 13 15h-2v-.5c0-1.1.45-2.1 1.17-2.83l1.24-1.26c.37-.36.59-.86.59-1.41 0-1.1-.9-2-2-2s-2 .9-2 2H8c0-2.21 1.79-4 4-4s4 1.79 4 4c0 .88-.36 1.68-.93 2.25"
              />
            </svg>
            <p
              class="MuiTypography-root MuiTypography-body1 css-rizt0-MuiTypography-root"
            >
              Disclaimer
            </p>
          </span>
          <span
            class="MuiAccordionSummary-expandIconWrapper css-1wqf3nl-MuiAccordionSummary-expandIconWrapper"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1umw9bq-MuiSvgIcon-root"
              data-testid="ExpandMoreIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M16.59 8.59 12 13.17 7.41 8.59 6 10l6 6 6-6z"
              />
            </svg>
          </span>
        </button>
      </h3>
      <div
        class="MuiCollapse-root MuiCollapse-vertical MuiCollapse-hidden css-cwrbtg-MuiCollapse-root"
        style="min-height: 0px;"
      >
        <div
          class="MuiCollapse-wrapper M

Error: TestingLibraryElementError: Unable to find an element by: [data-testid="toggle-sort"]

Ignored nodes: comments, script, style
<body>
  <div>
    <div
      class="MuiBox-root css-au904d"
    >
      <button
        class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-sizeMedium MuiButton-colorPrimary css-1blvszr-MuiButtonBase-root-MuiButton-root"
        tabindex="0"
        type="button"
      >
        <span
          class="MuiButton-icon MuiButton-startIcon css-1lxyhll-MuiButton-startIcon"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1umw9bq-MuiSvgIcon-root"
            data-testid="SwapVertIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M16 17.01V10h-2v7.01h-3L15 21l4-3.99zM9 3 5 6.99h3V14h2V6.99h3z"
            />
          </svg>
        </span>
      </button>
    </div>
    <div
      class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiAccordion-root MuiAccordion-rounded MuiAccordion-gutters css-1808mag-MuiPaper-root-MuiAccordion-root"
      style="--Paper-shadow: 0px 2px 1px -1px rgba(0,0,0,0.2),0px 1px 1px 0px rgba(0,0,0,0.14),0px 1px 3px 0px rgba(0,0,0,0.12);"
    >
      <h3
        class="MuiAccordion-heading css-cy7rkm-MuiAccordion-heading"
      >
        <button
          aria-controls="tier-list-disclaimer-content"
          aria-expanded="false"
          class="MuiButtonBase-root MuiAccordionSummary-root MuiAccordionSummary-gutters css-1f1pi2l-MuiButtonBase-root-MuiAccordionSummary-root"
          id="tier-list-disclaimer-header"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiAccordionSummary-content css-yfrx4k-MuiAccordionSummary-content"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-colorInfo MuiSvgIcon-fontSizeMedium css-kswx6x-MuiSvgIcon-root"
              data-testid="HelpIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2m1 17h-2v-2h2zm2.07-7.75-.9.92C13.45 12.9 13 13.5 13 15h-2v-.5c0-1.1.45-2.1 1.17-2.83l1.24-1.26c.37-.36.59-.86.59-1.41 0-1.1-.9-2-2-2s-2 .9-2 2H8c0-2.21 1.79-4 4-4s4 1.79 4 4c0 .88-.36 1.68-.93 2.25"
              />
            </svg>
            <p
              class="MuiTypography-root MuiTypography-body1 css-rizt0-MuiTypography-root"
            >
              Disclaimer
            </p>
          </span>
          <span
            class="MuiAccordionSummary-expandIconWrapper css-1wqf3nl-MuiAccordionSummary-expandIconWrapper"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1umw9bq-MuiSvgIcon-root"
              data-testid="ExpandMoreIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M16.59 8.59 12 13.17 7.41 8.59 6 10l6 6 6-6z"
              />
            </svg>
          </span>
        </button>
      </h3>
      <div
        class="MuiCollapse-root MuiCollapse-vertical MuiCollapse-hidden css-cwrbtg-MuiCollapse-root"
        style="min-height: 0px;"
      >
        <div
          class="MuiCollapse-wrapper MuiCollapse-vertical css-1x6hinx-MuiCollapse-wrapper"
        >
          <div
            class="MuiCollapse-wrapperInner MuiCollapse-vertical css-1i4ywhz-MuiCollapse-wrapperInner"
          >
            <div
              aria-labelledby="tier-list-disclaimer-header"
              class="MuiAccordion-region css-at4ued-MuiAccordion-region"
              id="tier-list-disclaimer-content"
              role="region"
            >
              <div
                class="MuiAccordionDetails-root css-1lj39kh-MuiAccordionDetails-root"
              >
                <p
                  class="MuiTypography-root MuiTypography-body1 css-rizt0-MuiTypography-root"
                >
     

Error: TestingLibraryElementError: Unable to find an element by: [data-testid="toggle-sort"]

Ignored nodes: comments, script, style
<body>
  <div>
    <div
      class="MuiBox-root css-au904d"
    >
      <button
        class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-sizeMedium MuiButton-colorPrimary css-1blvszr-MuiButtonBase-root-MuiButton-root"
        tabindex="0"
        type="button"
      >
        <span
          class="MuiButton-icon MuiButton-startIcon css-1lxyhll-MuiButton-startIcon"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1umw9bq-MuiSvgIcon-root"
            data-testid="SwapVertIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M16 17.01V10h-2v7.01h-3L15 21l4-3.99zM9 3 5 6.99h3V14h2V6.99h3z"
            />
          </svg>
        </span>
      </button>
    </div>
    <div
      class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiAccordion-root MuiAccordion-rounded MuiAccordion-gutters css-1808mag-MuiPaper-root-MuiAccordion-root"
      style="--Paper-shadow: 0px 2px 1px -1px rgba(0,0,0,0.2),0px 1px 1px 0px rgba(0,0,0,0.14),0px 1px 3px 0px rgba(0,0,0,0.12);"
    >
      <h3
        class="MuiAccordion-heading css-cy7rkm-MuiAccordion-heading"
      >
        <button
          aria-controls="tier-list-disclaimer-content"
          aria-expanded="false"
          class="MuiButtonBase-root MuiAccordionSummary-root MuiAccordionSummary-gutters css-1f1pi2l-MuiButtonBase-root-MuiAccordionSummary-root"
          id="tier-list-disclaimer-header"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiAccordionSummary-content css-yfrx4k-MuiAccordionSummary-content"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-colorInfo MuiSvgIcon-fontSizeMedium css-kswx6x-MuiSvgIcon-root"
              data-testid="HelpIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2m1 17h-2v-2h2zm2.07-7.75-.9.92C13.45 12.9 13 13.5 13 15h-2v-.5c0-1.1.45-2.1 1.17-2.83l1.24-1.26c.37-.36.59-.86.59-1.41 0-1.1-.9-2-2-2s-2 .9-2 2H8c0-2.21 1.79-4 4-4s4 1.79 4 4c0 .88-.36 1.68-.93 2.25"
              />
            </svg>
            <p
              class="MuiTypography-root MuiTypography-body1 css-rizt0-MuiTypography-root"
            >
              Disclaimer
            </p>
          </span>
          <span
            class="MuiAccordionSummary-expandIconWrapper css-1wqf3nl-MuiAccordionSummary-expandIconWrapper"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1umw9bq-MuiSvgIcon-root"
              data-testid="ExpandMoreIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M16.59 8.59 12 13.17 7.41 8.59 6 10l6 6 6-6z"
              />
            </svg>
          </span>
        </button>
      </h3>
      <div
        class="MuiCollapse-root MuiCollapse-vertical MuiCollapse-hidden css-cwrbtg-MuiCollapse-root"
        style="min-height: 0px;"
      >
        <div
          class="MuiCollapse-wrapper MuiCollapse-vertical css-1x6hinx-MuiCollapse-wrapper"
        >
          <div
            class="MuiCollapse-wrapperInner MuiCollapse-vertical css-1i4ywhz-MuiCollapse-wrapperInner"
          >
            <div
              aria-labelledby="tier-list-disclaimer-header"
              class="MuiAccordion-region css-at4ued-MuiAccordion-region"
              id="tier-list-disclaimer-content"
              role="region"
            >
              <div
                class="MuiAccordionDetails-root css-1lj39kh-MuiAccordionDetails-root"
              >
                <p
                  class="MuiTypography-root MuiTypography-body1 css-rizt0-MuiTypography-root"
                >
     
Error: Process completed with exit code 1.

--- a/src/components/tierList/TierListControls.tsx
+++ b/src/components/tierList/TierListControls.tsx
@@ -13,6 +13,7 @@ export function TierListControls({ onToggleSort }: TierListControlsProps) {
     return (
         <Box sx={{ display: 'flex', justifyContent: 'flex-end', mb: 3 }}>
             <Button 
+                data-testid="toggle-sort"
                 variant="outlined" 
                 onClick={onToggleSort}
                 startIcon={<SwapVertIcon />}

